frontend/src/features/mapSideBar/SidebarContents/subforms/BuildingReviewPage.test.tsx

```diff
@@ -25,17 +25,115 @@ const lCodes = {
 
 const store = mockStore({
   [reducerTypes.LOOKUP_CODE]: lCodes,
-  [reducerTypes.PARCEL]: { parcels: [] },
+  [reducerTypes.PROPERTIES]: { parcels: [] },
 });
 
+export const mockDetails = [
```

Comment:
No change required for this PR.
I see these types of mock datasets everywhere.  I think we probably should come up with a strategy to put them all in one place and make them generic/modifiable.